app/models/parser/twitter_search_item.rb

```diff
@@ -0,0 +1,36 @@
+module Parser
+  class TwitterSearchItem < Base
+    include ProviderTwitter
+
+    TWITTER_SEARCH_ITEM_URL = /https:\/\/twitter\.com\/search[^\s"]*/
+
+    class << self
+      def type
+        'twitter_search_item'.freeze
+      end
+
+      def patterns
+        [TWITTER_SEARCH_ITEM_URL]
+      end
+    end
+
+    private
+
+    # Main function for class
+    def parse_data_for_parser(_doc, _original_url, _jsonld_array)
+      handle_exceptions(StandardError) do
+        params = URI.parse(url).query.split('&').map { |pair| pair.split('=') }.to_h
+        src = params['src']
```

Comment:
Comment: We could ignore `src`, but doesn't hurt to keep it.

Reply:
Thank you, done!